src/TestRunner.js

```diff
@@ -285,7 +285,9 @@ class TestRunner {
       }
     }
 
-    testPaths = this._sortTests(testPaths);
+    if (testPaths.length > 1) {
+      testPaths = this._sortTests(testPaths);
```

Comment:
so one side-effect of `sortTests` is that it prepopulates `this._testPerformanceCache` which is later re-used in `_cacheTestResults`. If you want to skip this call, you have to re-factor the `JSON.parse` we do in sortTests into a separate function that `_cacheTestResults` can re-use. Otherwise we blow away the entire cache when we run a single test – ideally on a single test run we'll update the cache with the performance numbers of the single test for later, longer test runs.